packages/caver-klay/caver-klay-accounts/src/index.js

```diff
@@ -315,7 +315,16 @@ Accounts.prototype.signTransactionWithSignature = function signTransactionWithSi
  */
 Accounts.prototype.recoverTransaction = function recoverTransaction(rawTx) {
     var values = RLP.decode(rawTx);
-    var signature = Account.encodeSignature(values.slice(6,9));
+
+    // If the leading zero is trimmed, it will be filled with a valid length of '0'.
+    const arr = values.slice(7,9).map((sig) => {
```

Comment:
This slicing works only if the raw Tx is a legacy transaction. Is this okay?

Reply:
oh, i did't think about other transaction type.
original recoverTransaction logic cannot handle other transaction type ;-;... 
This has to be implemented with various transaction type... ! 
I will implement that !! Thank you for reviewing ;-;..

Reply:
@kjhman21 Error and test code is added :) Please take a look !